sonar-scanner-engine/src/main/java/org/sonar/batch/report/TestExecutionAndCoveragePublisher.java

```diff
@@ -113,7 +113,7 @@ public TestExecutionAndCoveragePublisher(BatchComponentCache resourceCache, Test
   }
 
   @Override
-  public void publish(BatchReportWriter writer) {
+  public void publish(ScannerReportWriter writer) {
     for (final BatchComponent component : componentCache.all()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the total number of break and continue statements in this loop to use at most one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS135)